### ISSUE TITLE
Remap in Point blank X

### ISSUE THREAD

**Issue by bromlecornu** [argonlefou/DemulShooter#266]
Hi everyone

 

I need your help with my remapping key for AHK

 

this is my script

 

```
#SingleInstance Force


Run, E:\LaunchBox\DemulShooter\DemulShooter.exe -target=es4 -rom=pblankx

Run,PBX100-2-NA-MPR0-A63.exe

5::MButton

Esc::
Process,Close,PBX100-2-NA-MPR0-A63.exe
Run,taskkill /im "PBX100-2-NA-MPR0-A63.exe" /F

Run,taskkill /im "DemulShooter.exe" /F
ExitApp
return
```
 

I want to remap 5 (credit) to middle mouse button (rear left of my sinden) it doesn't work

 

Any idea ?

 

Thx

**Comment by argonlefou** [comment]
Hello,

DemulShooter is getting input events directly from the device, so sending `MButton` event with AHK won't do anything as the device will not send anything itself.

If you want to achieve what you're trying to do, you'll need to enable virtual mouse buttons for your Sinden in `Demulshooter_GUI.exe` and set the middle mouse key to `5`

Then you'll need to look on the Sinden configuration side (a part I'm not familiar with, so I don't know if it's possible) so that your physical rear button will send the 5 Keyboard Key instead of the Middle click

In a nutshell :

`Sinden Button press` --> `Sinden software sends 5` --> `DemulShooter detects 5 keypress` --> `Demulshooter detects virtual Middle mouse button`
